src/inference/src/ie_network_reader.hpp

```diff
@@ -10,6 +10,7 @@
 #include "ie_blob.h"
 #include "ie_iextension.h"
 #include "openvino/core/extension.hpp"
+#include "openvino/frontend/manager.hpp"
```

Comment:
This include file is not used in the header, can you remove it or move it to the cpp file if it's required there?

Reply:
sure

Reply:
ok